createdisk.sh

```diff
@@ -228,7 +228,7 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl stopp $(sudo crict
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl rmp $(sudo crictl pods -q)'
```

Comment:
Maybe commit log should be "We are not using sudo to remove the `config.json` file from `/etc/kubernetes`. This silently fails, and the resulting bundle contains this file."

Reply:
commit log does not seem to have been adjusted in the latest version?

Reply:
@cfergeau it's updated :) https://github.com/code-ready/snc/pull/71/commits/4d78be025616ae861e60bf22c2439ac9f6bde4e4